### ISSUE TITLE
OO Shutup Tweak not working - Responding with '404 (Not Found)' Status Code

### ISSUE THREAD

**Issue by og-mrk** [ChrisTitusTech/winutil#1853]
**Describe the bug**
The Tweak `O&O` Shutup is not working, responding with `404 (Not Found)`

**To Reproduce**
1. Open a PowerShell Terminal as Admin
2. Run `iex "& { $(irm christitus.com/win) } -Debug"` to run WinUtil with the Debug Switch (To See More Info outputted in the PowerShell Terminal)
3. Goto Tweaks Tab in WinUtil, and Only Tick the `Run OO Shutup` CheckBox (Please note that O&O Shutup doesn't work as well when selecting the other tweaks, or all of them)
4. See the following (Or similar) output in PowerShell Terminal:
```powershell
DEBUG: WinUtil lost focus
DEBUG: Getting checkboxes to set 367
DEBUG: WinUtil lost focus
DEBUG: Adding: WPFTweaksOO under: WPFTweaks
DEBUG: Number of tweaks to process: 1
DEBUG: Inside Number of tweaks to process: 1
DEBUG: This is a tweak to run WPFTweaksOO count: 0
DEBUG: Tweaks: WPFTweaksOO
DEBUG: Invoke-WPFOOSU -action "recommended" and state is
Running Script for WPFTweaksOO
WARNING: A runtime exception occurred.
WARNING: Response status code does not indicate success: 404 (Not Found).
=================================
--     Tweaks are Finished    ---
=================================
```

**Expected behavior**
The Tweak Runs and successfully does what it states it will do according to the Tooltip (description) of `Run OO Shutup` Checkbox Tweak.

**Screenshots**
![image](https://github.com/ChrisTitusTech/winutil/assets/70659536/2163c3a6-e441-46a1-be69-749ef1b2d57a)

**Additional context**
![image](https://github.com/ChrisTitusTech/winutil/assets/70659536/dd29d576-f67b-488c-9298-eff3fab3a692)

**Comment by ChrisTitusTech** [comment]
Thanks OG, I'll take a look at what happened with O&O.

**Comment by ChrisTitusTech** [comment]
Fixed in c8ae4a8
